drivers/counter/Kconfig.mspm0gxxxx

```
@@ -0,0 +1,5 @@
+config COUNTER_MSPM0GXXXX
+    bool "Driver to interface with MSPM0G hardware clocks"
+    default y
```

We should not enforce drivers on this level.
```suggestion
    default n
```